.github/workflows/pull_request.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       - name: Make
         run: |
           ccache -z
-          cmake --build build/ -j $(nproc)
+          cmake --build build/ -j $(($(nproc)/2+1))
```

Comment:
This will slow down the whole compile steps.

Reply:
> This will slow down the whole compile steps.

yes, I see 137 system error returned at compile time, which means out of memory, so I need to slow compilation to ensure stability

Reply:
> This will slow down the whole compile steps.

Let's look a few more times to find an optimal solution.

Reply:
We can try to change the runner's resources to alleviate this problem.

Reply:
> We can try to change the runner's resources to alleviate this problem.

Good idea. let me undo that restriction.